metadata/src/main/java/org/apache/kafka/image/FeaturesImage.java

```diff
@@ -46,6 +46,18 @@ public final class FeaturesImage {
         ZkMigrationState.NONE
     );
 
+    public static final FeaturesImage LATEST = new FeaturesImage(
```

Comment:
Hmm... I'd rather not add `FeaturesImage.LATEST` here since it feels like something that is only for testing. Can we just add this to `ReplicaManagerTest.scala`, for now?

LGTM once that's addressed

Reply:
Done!